core/assisted-service/01-provisioningoverride.yaml

```diff
@@ -9,3 +9,5 @@ metadata:
 spec:
   watchAllNamespaces: true
   provisioningNetwork: Disabled
+  ### Remove comments if you want to disable TLS verification
```

Comment:
```suggestion
  ### We add provisioing-configuration under the management of Argo CD to  
  ### selectively utilize disableVirtualMediaTLS: true upon user needs.
  ### This is because some baremetal systems need HTTP (disable TLS==true).
  ### Note that if the user disables TLS verification the user will need
  ### commit the manifest changes to get tracked by Argo CD and review  
  ### for potential changes (e.g., different branch) in the default ArgoCD 
  ### AppSet existent in this repo leveraging this manifest. 
  ### By default keep TLS enabled for the Virtual Media Server.
  ### Remove comments if you want to disable TLS verification
```

Reply:
Added comments